cmd/harmony/bls.go

```diff
@@ -25,7 +27,19 @@ func setupConsensusKeys(hc harmonyConfig, config *nodeconfig.ConfigType) multibl
 			os.Exit(100)
 		}
 	})
-	config.ConsensusPriKey = multiBLSPriKey
+
+	// Dedup private keys
+	uniqueKeys := map[bls.SerializedPublicKey]struct{}{}
+	deduped := multibls.PrivateKeys{}
```

Comment:
I would suggest to use 

```
deduped := make(multibls.PrivateKeys, 0, len(multiBLSPriKey))
```

to avoid double expansion. Also empty slice declaration via literal is not a good coding practice. [Reference](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)

Reply:
good point done.